__tests__/helpers/db-helpers.js

```diff
@@ -1,44 +1,40 @@
-// We use requireActual here in case the test calling it has mocked any
-// of these modules
-const path = require.requireActual('path');
-const fs = require.requireActual('fs');
-const JSON5 = require.requireActual('json5');
-const knex = require.requireActual('knex');
-const mysql = require.requireActual('mysql');
-const _ = require.requireActual('lodash');
+// Be careful not to mock any of these in your test!
+import path from 'path';
+import fs from 'fs';
+import JSON5 from 'json5';
+import knex from 'knex';
+import _ from 'lodash';
 
 const databaseConnectionJSON5String = fs.readFileSync(
   path.join(__dirname, '../../config/database.config.json5'),
 );
 
 const databaseConnectionConfig = JSON5.parse(databaseConnectionJSON5String);
 
-export const getDatabaseConnection = databaseName =>
-  knex({
+export const getDatabaseConnection = databaseName => {
```

Comment:
I was just experimenting with different things to see what would work well but I think this is probably cleaner code anyway, so I'm keeping it :)